src/components/shared/forms/pill.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Svg } from 'components/shared/svg/svg';
+import { joinClasses } from 'utilities';
+
+import './styles/pill.css';
+
+export const Pill = (props) => {
+  const { svg, label, onSvgClick} = props;
```

Comment:
Since you now longer need the `this`, you can do the object destructuring assignment in the function definition itself.
```
export const Pill = ({ svg, label, onSvgClick }) => (
  <div className .... 
  </div>
);
```

Reply:
Changed.